tools/automation/package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "built-in-extensions-update-automation",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Scripts for updating, adding, removing, and building extensions in che-theia",
+  "main": "check-extension-updates.js",
+  "scripts": {
+    "build": "yarn run format && yarn run compile && yarn run lint",
+    "compile": "tsc --project .",
+    "format": "prettier --write **/*.ts **/**/*.ts package.json",
```

Comment:
it seems that code is not formatted b/c when I run the command there are changes